site/assets/scss/_sidebar.scss

```diff
@@ -36,18 +53,16 @@
   padding: .1875rem .5rem;
   margin-top: .125rem;
   margin-left: 1.375rem; // Boosted mod: changed value
-  color: rgba($black, .65);
+  color: var(--bs-body-color);
```

Comment:
```suggestion
  color: var(--bs-secondary-color);
```

Reply:
For this one, I don't remember if we had a specific design for Boosted somewhere but I found it rather nice in black